docs/Dialects/Seq/RationaleSeq.md

```diff
@@ -226,34 +226,27 @@ merging, read/write conflicts, etc.) and may serve as a target for other
 high-level abstractions.
 
 The high-level memory abstraction is split into two parts:
-* Memory *allocation* is defined by the `seq.hlmem` operation. This operation defines
-the inner workings of the memory. This includes:
-  - number of read/write ports
-  - read/write latency
-  - Memory size and inner data type
-  The `seq.hlmem` will define references to the read- and write ports of the
-  allocated memory.
-- Memory *access* is defined by separate operations which unwraps a memory port reference
-to its structural components. Port access operations are defined at the same
-level of abstraction as the core RTL dialects and  contain no notion of control
-flow.
+- Memory *allocation* is defined by the `seq.hlmem` operation. This operation
+defines the internal memory structure. For now, this strictly pertains to the
+layout of the memory (dimensionality) and element type.
+- Memory *access* is defined by separate port operations which reference the
+allocated memory. Port access operations are defined at the same level of
+abstraction as the core RTL dialects and contain no notion of control
+flow. As such, for e.g. a write port with a non-zero latency, the encapsulating
+IR must already have accounted for this latency.
```

Comment:
Should we add a note about conflicting writes? 'The behavior of conflicting writes is defined by the lowering. Generally speaking, it should be considered as undefined.'